prime/src/components/borrow/actions/UniswapSwapActionCard.tsx

```diff
@@ -83,11 +79,11 @@ export default function UniswapSwapActionCard(props: ActionCardProps) {
       newAmountOutMin = truncateDecimals(newAmountOutMin, tokenTypeOutDecimals);
     }
 
-    const parsedAmountIn = parseFloat(newAmountInExact) || 0;
-    const parsedAmountOut = parseFloat(newAmountOutMin) || 0;
+    const parsedAmountIn = GN.fromDecimalString(newAmountInExact || '0', tokenTypeInDecimals);
```

Comment:
You did this same conversion in-line. Could move variable definition up? Honestly I could go either way on this.

Reply:
Addressed in latest PR